.env.sample

```diff
@@ -4,20 +4,22 @@ NODE_HOSTNAME=
 # Deployment env vars
 NETWORK_NAME=testnet-beta
 POKTROLLD_IMAGE_REPO=ghcr.io/pokt-network/poktrolld
+PATH_GATEWAY_IMAGE_REPO=ghcr.io/buildwithgrove/path
 
-# The version of the image to run off-chain components such as RelayMiner and AppGate.
+# The version of the image to run off-chain components such as RelayMiner and PATH.
 # Adjust the value when need to upgrade off-chain actors.
 POKTROLLD_IMAGE_TAG_OFF_CHAIN=0.0.10 # latest
+PATH_GATEWAY_IMAGE_TAG=main
 POKTROLLD_LOG_LEVEL=info
 
 # Credentials for AppGateServer, Gateway and RelayMiner
```

Comment:
Still seeing `AppGateServer` come up in a few places. 

Also in the `.gitignore`, `network-overview`, prometheus.yml, etc...

Are there any others? Could you double check and update this PR?

Reply:
Left some occurrences of `AppGate` with a `TODO` until `PATH` metrics are added.